pyccl/core.py

```diff
@@ -19,7 +20,8 @@
     'bbks':             lib.bbks,
     'boltzmann_class':  lib.boltzmann_class,
     'boltzmann_camb':   lib.boltzmann_camb,
-    'boltzmann_isitgr':   lib.boltzmann_isitgr,
+    'pklin_from_input': lib.pklin_from_input,
```

Comment:
This is part of the public API.

Reply:
Ah, what are the necessary changes so this is not the case?

Reply:
nothing new appears in the python api that we wouldn't take out later.

Reply:
OK @chrgeorgiou let's do the following.
1. Remove this line.
2. (more instructions below)